src/packages/shared-utils/package-actions/getAvailableActions.ts

```diff
@@ -11,7 +11,9 @@ export const getAvailableActions = (
   result: opensearch.main.Document
 ) => {
   const checks = PackageCheck(result);
-  return checks.isSpa
-    ? rules.filter((r) => r.check(checks, user)).map((r) => r.action)
-    : [];
+  return [
+    ...((checks.isWaiver || checks.isSpa)
```

Comment:
This can be done inside the `ActionRule`s for closer control. We should move away from conditionally outputting outside of those rules

Reply:
Also unsure why we're spreading an array inside an array